src/main/play-doc/api/BundleAPI.md

```diff
@@ -61,6 +61,150 @@ Returns a list of host and port of the running service given a service name.
 ### Request
 
 ```
+GET {SERVICE_LOCATOR}/v2/service-hosts/{service-name}
+```
+
+Field            | Description
+-----------------|------------
+SERVICE\_LOCATOR | The environment variable value of the same name. This environment variable translates to an http address e.g. `http://10.0.1.22:9008` given that ConductR is running on `10.0.1.22` with a service locator port bound to 9008.
+service-name     | The name of the service required and expressed as a path e.g. `/customers`.
+
+#### Success
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "protocol":"http",
+    "ipv4Host":"10.11.23.22",
+    "port":7001
+  },
+  {
+    "protocol":"http",
+    "ipv4Host":"10.11.23.22",
+    "port":7002
+  }
+]
+```
+
+Where `10.11.23.22:7001` and `10.11.23.22:7002` are the host and host port of the service which is currently running. The following table shows the complete list of fields available to a reply:
+
+Field            | Description
```

Comment:
Can we order the fields either alphabetically or sorted by required and optional? I'd add the information to a field if it is required. So far we only describe that a given field is optional.